src/mavsdk_server/src/mavsdk_server_api.h

```diff
@@ -16,6 +16,7 @@ extern "C" {
 #define DLLExport __attribute__((visibility("default")))
 #endif
 
+    struct MavsdkServer;
```

Comment:
```suggestion
struct MavsdkServer;

```

Just a nitpick, for the style check :laughing:

Reply:
👍